src/main/java/workflow/state/ErrorState.java

```diff
@@ -15,4 +13,8 @@ public boolean isWarning() {
     public boolean isReady() {
         return false;
     }
+
+    public String toString() {
```

Comment:
we can likely remove this, it was for debugging purposes, there isn't anything inherantly nasty about overriding toString, every Object has one implicitly.